release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2154,6 +2154,20 @@ To make the required change, follow the instructions in link:https://access.redh
 
 * In hosted control planes for {product-title}, on the bare metal and {VirtProductName} platforms, the auto-repair function is disabled. (link:https://issues.redhat.com/browse/OCPBUGS-20028[*OCPBUGS-20028*])
 
+[id="ocp-telco-ran-4-14-known-issues"]
+* When you run CNF latency tests on a {product-title} cluster, the `oslat` test can sometimes return results greater than 20 microseconds. This results in a `oslat` test failure.
++
+(link:https://issues.redhat.com/browse/RHEL-9279[*RHEL-9279*])
+
+* When you use `preempt-rt` patches with the realtime kernel and you update the SMP affinity of a network interrupt, the corresponding IRQ thread doesn't immediately receive the update.
+Instead, the update takes effect when the next interrupt is received, and the thread is subsequently migrated to the correct core.
++
+(link:https://issues.redhat.com/browse/RHEL-9148[*RHEL-9148*])
+
+* Low-latency applications that rely on high-resolution timers to wake up their threads might experience higher wake up latencies than expected. Although the expected wake up latency is under 20us, latencies exceeding this can occasionally be seen when running the cyclictest tool for long durations (24 hours or more). Testing has shown that wake up latencies are under 20us for over 99.999999% of the samples.
```

Comment:
And this one too - @linuxqiao can you provide lgtm on this OCP 4.14 known issue description for https://issues.redhat.com/browse/RHELPLAN-138733 please

Reply:
Known issue, LGTM.